session/proxy_examples.rst

```diff
@@ -110,12 +110,12 @@ and decrypt the session as required::
         {
             $data = parent::read($id);
 
-            return mcrypt_decrypt(\MCRYPT_3DES, $this->key, $data);
+            return openssl_decrypt($data, \OPENSSL_ALGO_SHA1, $this->key);
```

Comment:
Don't we really have a choice to solve this problem using pure PHP code (no libraries)? Let's remember that we're "just" encryption some "silly" session data, not the nuclear launch codes. Thanks!